### ISSUE TITLE
extra --add-registry with no registry...

### ISSUE THREAD

**Issue by thoraxe** [openshift/openshift-ansible#514]
Installation fails because Docker won't start. Looking at `/etc/sysconfig/docker` you find:

```
ADD_REGISTRY='--add-registry registry.access.redhat.com --add-registry '
```

**Comment by thoraxe** [comment]
The problem is in this task:

https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_node/tasks/main.yml#L83

**Comment by wshearn** [comment]
Merged #516 should be fixed now
